template/meta_refresh.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+    <meta http-equiv="refresh" content="0; url={{ refresh_url }}/confirmredirect/{{pk}}?whatsappid={{whatsappid}}" />
```

Comment:
We should rather use the url template tag for building urls in templates: https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#url